modules/core/src/main/scala/org/tessellation/infrastructure/trust/storage/TrustStorage.scala

```diff
@@ -113,6 +113,8 @@ object TrustStorage {
             .toMap
         }
       }
+        .map(_.getOrElse(Map.empty[PeerId, Double]))
```

Comment:
It looks like there's no longer a distinction `seedlist` being `None` vs `Some(empty set)`
If so we should refactor

Reply:
Changed the seedlist to `Set[SeedlistEntry]` instead of `Option[Set[SeedlistEntry]]`.